Domain/src/main/java/starterProject/domain/Car.java

```diff
@@ -1,10 +1,13 @@
 package starterProject.domain;
+import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 import java.util.ArrayList;
 
-@Data @Builder public class Car {
+@Data @NoArgsConstructor @AllArgsConstructor
```

Comment:
I don't believe we need the `@NoArgsConstructor @AllArgsConstructor` as aren't we using the Builder to instantiate objects. Maybe just the `@AllArgs` one. 

If we have a no argument constructor, when we initialise a class what are the values of our member variables / fields. If we try to do anything without having values assigned to them we will get errors.

Reply:
Can you make sure that we are using the least amount of annotations possible to make it work?